services/auth/source/db/source.go

```diff
@@ -18,7 +18,7 @@ func (source *Source) FromDB(bs []byte) error {
 	return nil
 }
 
-// ToDB exports an SMTPConfig to a serialized format.
+// ToDB exports an DBConfig to a serialized format.
```

Comment:
I think there is no `DBConfig` in code.

This method is just a dummy one and does nothing.

```suggestion
// ToDB exports the config to a byte slice to be saved into database (this method is just dummy and does nothing for DB source)
```

Reply:
Or just remove it. These meaningless comments are from the time, the linter enforced comments.
The comment from `FromDB` is wrong too.